Mathlib/Order/Interval/Finset/Fin.lean

```diff
@@ -89,6 +89,8 @@ theorem attachFin_Ioo_eq_Ioi : attachFin (Ioo a n) (fun _x hx ↦ (mem_Ioo.mp hx
 
 section deprecated
 
+set_option linter.deprecated false
```

Comment:
Kim wants you to write `set_option linter.deprecated false in` on each declaration separately